src/ontology/mondo-edit.properties

```diff
@@ -0,0 +1,5 @@
+#Tue Jan 04 12:54:28 PST 2022
```

Comment:
did you mean to edit this file?

Reply:
no I didn't... I don't know how it was edited in the first place...

Reply:
fixed